### PR TITLE
add log's dependency for bookkeeper-dist-server

### PR DESCRIPTION
**Pull request by StevenLuMT**
Descriptions of the changes in this PR:

### Motivation

like pulsar's distribution/server
add log's dependency in bookkeeper-dist/server 

### Changes

1.  add log's dependency in bookkeeper-dist/server/pom.xml
2. then auto add log's dependency to bookkeeper-server-*.*.*-SNAPSHOT-bin.tar.gz

### CODE REVIEW

**Comment by StevenLuMT** [comment]
rerun failure checks

File: bookkeeper-dist/server/pom.xml [hunk 1]
```diff
@@ -31,7 +31,80 @@
   <packaging>jar</packaging>
   <name>Apache BookKeeper :: Dist (Server)</name>
 
+  <properties>
+    <disruptor.version>3.4.0</disruptor.version>
```
Comment (eolivelli) [inline bookkeeper-dist/server/pom.xml:35]: this should be here

also, why do you want to add this dependency ?

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: bookkeeper-dist/server/pom.xml (hunk 1 above)
Comment (StevenLuMT) [reply bookkeeper-dist/server/pom.xml:35]: @eolivelli 
log4j-core need disruptor to write log, so import disruptor like pulsar
image

**Comment by Shoothzj** [comment]
I am confused too. why we need this dependency ? We can print log well for now.
Do you want to support Async log on bookkeeper?

cc @eolivelli

**Comment by StevenLuMT** [comment]
> Do you want to support Async log on bookkeeper?

yes, this pr's motivation is to support async log  @Shoothzj

**Review by Shoothzj — CHANGES_REQUESTED** [review]
Why  we need add `log4j-1.2-api`  dependency?

**Comment by StevenLuMT** [comment]
rerun failure checks

**Comment by StevenLuMT** [comment]
> Why we need add `log4j-1.2-api` dependency?

 @Shoothzj 
Legacy Log4j 1.2 API , it's the dependency in org.apache.logging.log4j.log4j-bom

image

**Comment by Shoothzj** [comment]
@StevenLuMT of course it's defined in the bom, but it's not the reason we put it in our dependency! I didn't see any usage of log4j1 in bookkeeper.

**Comment by StevenLuMT** [comment]
> @StevenLuMT of course it's defined in the bom, but it's not the reason we put it in our dependency! I didn't see any usage of log4j1 in bookkeeper.

good catch, log4j-1.2-api is no use in project, 
I misunderstood,thanks very much @Shoothzj

**Comment by shoothzj** [comment]
closed by #4269 thanks for your initiation contribution.

**Pull request closed without merging**